### PR TITLE
feat: new ui streaming views

### PR DESCRIPTION
**Pull request by mwill8886**
resolves: COIL-1570

The new UI is still not visible to the public and will be behind a feature flag
Added the streaming views to the new extension UI.
Streaming View States: 
- Web monetized site
- Not web monetized site
- User with no membership
- Coil site
- Coil site, discover page

### CODE REVIEW

File: packages/coil-extension/src/popup/contants.ts [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
   streaming = 'streaming',
-  streamingNoWebMo = 'streaming/notmonetized',
-  streamingNoMembership = 'streaming/nomembership',
-  streamingCoil = 'streaming/coil',
-  streamingCoilDiscover = 'streaming/coil/discover',
+  // streamingWebMo = 'streaming/monetized',
+  // streamingNoWebMo = 'streaming/notmonetized',
+  // streamingNoMembership = 'streaming/nomembership',
+  // streamingCoil = 'streaming/coil',
+  // streamingCoilDiscover = 'streaming/coil/discover',
```
Comment (bboynton97) [inline packages/coil-extension/src/popup/contants.ts:7]: Did you mean to keep these commented out?

File: packages/coil-extension/src/popup/mocks/loadMockedStates.tsx [hunk 2]
```diff
@@ -146,7 +146,8 @@ const userCanTip = mockState({
 
 const userCanTipNewUi = mockState({
   monetized: true,
-  coilSite: 'https://coil.com/p/mirrae/Letter-to-my-Daughter/-ZInTJqN-',
```
Comment (bboynton97) [inline packages/coil-extension/src/popup/mocks/loadMockedStates.tsx:149]: Same here

**Review by bboynton97 — CHANGES_REQUESTED** [review]

File: packages/coil-extension/src/popup/mocks/loadMockedStates.tsx (hunk 2 above)
Comment (mwill8886) [reply packages/coil-extension/src/popup/mocks/loadMockedStates.tsx:149]: This was intentional, `loadMockedState` is the mocked data for the view previewer. That's essentially what I use to build the views.

File: packages/coil-extension/src/popup/contants.ts (hunk 1 above)
Comment (mwill8886) [reply packages/coil-extension/src/popup/contants.ts:7]: Nope, and I just realized the file is misspelled

File: packages/coil-extension/src/popup/components/views/Router.tsx [hunk 3]
```diff
@@ -38,70 +46,46 @@ export const Router = () => {
...
-            no webmo
-          </button>
-        </NewHeaderFooterLayout>
-      )
+      // check the logic for what state to display for the streaming page
+
+      // Streaming - No Membership
+      if (
+        !user?.subscription ||
+        (user.subscription && !user.subscription.active)
```
Comment (sublimator) [inline packages/coil-extension/src/popup/components/views/Router.tsx:66]: Could be condensed:
!user?.subscription?.active

File: packages/coil-extension/src/popup/components/NavBar.tsx [hunk 4]
```diff
@@ -3,7 +3,7 @@ import { Box, styled, Theme } from '@material-ui/core'
 import SettingsIcon from '@material-ui/icons/Settings'
 
 import { useRouter } from '../context/routerContext'
-import { ROUTES } from '../contants'
+import { ROUTES } from '../constants'
```
Comment (bboynton97) [inline packages/coil-extension/src/popup/components/NavBar.tsx:6]: love it

**Review by bboynton97 — APPROVED** [review]

**Review by chris-jackson-actionqa — APPROVED** [review]
Approving, but with reservations.  Not able to test, so not sure if things are correct.  But we need this checked in order to actually test it.  Catch 22.

**Pull request merged**